scripts/npm/link/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var path = require('path'),
+    fs = require('fs'),
+    componentsDir = path.join(__dirname, '..', '..', '..', 'src'),
+    components = getDirectories( componentsDir ),
+    exec = require( 'child-process-promise' ).exec;
```

Comment:
Trivial: Put var declarations on their own lines to avoid similarities to an array or object literal declaration.